Pod/Classes/SegmentifyManager.swift

```diff
@@ -811,7 +812,7 @@ public class SegmentifyManager : NSObject {
         let encodedData = try? JSONEncoder().encode(segmentifyObject)
         
         
-        let dataCenter  = SegmentifyManager.setup.dataCenterUrl
+        let dataCenter  = SegmentifyManager.setup.dataCenterUrl!.contains("dce1") ? SegmentifyManager.setup.dataCenterUrl : SegmentifyManager.pushDomain
```

Comment:
@ebruersoy neden dce1 mi diye kontrol ediyoruz? bir de gc müşterileri için gimli-eu olmalı, bu hali ile hep tr hesaplarını düşünüyoruz sanki.

Reply:
çünkü default u dce1 idi, dce1 i kullananlar da var diye anladım. sdk yı kullanan v1 hesabı yoksa silerim. eu için ekliyim haklısın eksik olmuş